docs/interfaces/sheet.html

```diff
@@ -117,7 +117,7 @@ <h3>name</h3>
 					<div class="tsd-signature tsd-kind-icon">name<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in SheetInterfaces.ts:147</li>
```

Comment:
Interesting-- Do we know why the relative path disappeared off this text?

Reply:
TypeDoc now adds the path from the root in the "Defined in" statement.